workspaces/website/src/pages/(components)/NavbarStickyBanner/NavbarStickyBanner.tsx

```diff
@@ -77,6 +79,7 @@ const NavbarStickyBanner = ({
             }}
             _hover={{ bgColor: "white" }}
             variant="solid"
+            target={openNewPage ? "_blank" : ""}
```

Comment:
Please fix according to docs:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target